browserstack.yml

```diff
@@ -0,0 +1,75 @@
+# =============================
+# Set BrowserStack Credentials
+# =============================
+# Add your BrowserStack userName and accessKey here or set BROWSERSTACK_USERNAME and
+# BROWSERSTACK_ACCESS_KEY as env variables
+userName: rohansingh_hI71fO
+accessKey: xcdy9uCMwBEB5b3HTH26
+
```

Comment:
Absolutly a no no to put your credentials in a repository. It is always a no but especially on Public repositories. 
Delete this and if possible regenerate new key.
There are people out there scanning GitHub for passwords. And since you committed this, this will always be there.

Reply:
Marked